x-pack/plugins/cases/server/client/cases/create.test.ts

```diff
@@ -702,4 +702,125 @@ describe('create', () => {
       ).rejects.toThrowErrorMatchingInlineSnapshot(`"No custom fields configured."`);
     });
   });
+
+  describe('throwIfMissingRequiredCustomField', () => {
+    const casesClient = createCasesClientMock();
+
+    beforeEach(() => {
+      jest.clearAllMocks();
+    });
+
+    it('does not throw if all required custom fields are in the request', async () => {
```

Comment:
To my understanding the tests do not test required fields but the optional fields. The `casesClient.configure.get` does not contain the `required` property.